### PR TITLE
docs(sem): how-tos

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: identity-and-access-management/secret-manager/how-to/create-secret.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [generated an API key](docs/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
+</Message>
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. The **Secret Manager** overview page displays.
+2. Click **Create a secret**.
+3. Configure your secret:
+    * Select the region in which you want to create your secret.
+    * Enter a name for your secret.
+    * Optionally, add tags.
```
Comment (farhakennar) [inline identity-and-access-management/secret-manager/how-to/create-secret.mdx:30]:
```suggestion
    * Add tags (optional)
```
So the sentence still starts with a verb

**Review by farhakennar — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [generated an API key](/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
+  - Your have [several secrets](/identity-and-access-management/secret-manager/how-to/create-secret/)
+</Message>
+
+## Filter your secrets by tags
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:26]: On ne met plus les "How-to" dans les titres ?

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
+2. In the **Tags** drop-down box type in the tag(s) you want to filter your secrets by.
+     <Lightbox src="scaleway-filter-secrets-tags.webp" alt="" />
+3. Click the **Apply filters** button. Your secrets with the required tag(s) display.
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:31]:
```suggestion
3. Click **Apply filters**. Your secrets with the required tag(s) display.
```

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+4. Click the **Reset filters** button to reset the filters.
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:32]:
```suggestion
4. Click **Reset filters**.
```

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
+2. In the **Region** drop-down box select the region you want to filter your secrets by.
+    <Message type="important">
+     - You can only select one region
+     - Regions you can filter your secrets by are `AMS`, `PAR` and `WAW`
+    </Message>
+3. Click the **Apply filters** button. The secrets created within the region you have selected display.
+     <Lightbox src="scaleway-filter-secrets-region.webp" alt="" />
+4. Click the **Reset filters** button to reset the filters.
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:44]: Ca fait un peu "click delete to delete"...same comment as above

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+## Filter your secrets by region
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
+2. In the **Region** drop-down box select the region you want to filter your secrets by.
+    <Message type="important">
+     - You can only select one region
+     - Regions you can filter your secrets by are `AMS`, `PAR` and `WAW`
+    </Message>
+3. Click the **Apply filters** button. The secrets created within the region you have selected display.
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:42]:
```suggestion
3. Click **Apply filters**. The secrets created in the region selected display.
```

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+     <Lightbox src="scaleway-filter-secrets-region.webp" alt="" />
+4. Click the **Reset filters** button to reset the filters.
+
+<Message type="note">
+  You can also filter your secrets by tags and region at the same time
```
Comment (ofranc) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:47]:
```suggestion
  You can also filter your secrets by tags and region simultaneously.
```

**Review by ofranc — APPROVED** [review]

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx (hunk 2 above)
Comment (nedaehm) [reply identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:26]: On les met quand y'a deux sortes de how-tos dans un seul, non ? Genre là y a "filter by tags" et juste après "filter by region"

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx (hunk 5 above)
Comment (nedaehm) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:44]:
```suggestion
4. Click **Reset filters** once you are done.
```

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx (hunk 2 above)
Comment (nedaehm) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:26]:
```suggestion
## How to filter your secrets by tags
```

File: identity-and-access-management/secret-manager/how-to/filter-secrets.mdx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+## Filter your secrets by tags
+
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
+2. In the **Tags** drop-down box type in the tag(s) you want to filter your secrets by.
+     <Lightbox src="scaleway-filter-secrets-tags.webp" alt="" />
+3. Click **Apply filters**. Your secrets with the required tag(s) display.
+4. Click **Reset filters** once you are done.
+
+## Filter your secrets by region
```
Comment (nedaehm) [inline identity-and-access-management/secret-manager/how-to/filter-secrets.mdx:34]:
```suggestion
## How to filter your secrets by region
```

**Pull request merged**